app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -185,6 +189,14 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
                 canShowUpinToast = false
             }
         })
+
+        pinnedTileViewModel.getTileList().observe(lifeCycleOwner, Observer {
+            if (it != null) {
```

Comment:
Why would the tile list value ever be null? If it's because it's loading, can we add a "Loading" state instead (e.g. with sealed classes) so it's clear what this means?

Reply:
Technically never. Old Java habit 😞 . Will update